quelpa.el

```diff
@@ -1806,6 +1806,22 @@ the `quelpa' command has been run in the current Emacs session."
                 (quelpa item)))
             quelpa-cache))))
 
+;;;###autoload
+(defun quelpa-upgrade (rcp)
+  "Upgrade a package found in `quelpa-cache'.
+RCP is a melpa recipe (list)."
+  (interactive
+   (list (when (quelpa-setup-p)
+           (let* ((quelpa-melpa-recipe-stores `(,quelpa-cache))
```

Comment:
Why the backquote and unquote here?

Reply:
`quelpa-melpa-recipe-stores` expect a list of alist.